src/com/company/Main.java

```diff
@@ -37,76 +47,158 @@ public static void main(String[] args) {
+        Date dateOfBirth = null;
+        String dateOfBirthInput = scanner.nextLine();
+        String dateOfBirthPattern = "dd/MM/yyyy";
+        SimpleDateFormat formatDateOfBirth = new SimpleDateFormat(dateOfBirthPattern);
+        try {
+            dateOfBirth = formatDateOfBirth.parse(dateOfBirthInput);
+        } catch (ParseException e) {
+            e.printStackTrace();
+            System.out.println("Enter integers only");
+        }
+
+        System.out.println("Enter your interests *** max symbols = 250 ***");
         char limit = 250;
-        String interest = scanner.nextLine();
+        String interests = scanner.nextLine();
+        if (interests.length() > limit) {
+            interests = interests.substring(0, limit);
+        }
 
-        if(interest.length() > limit) {
-            interest = interest.substring(0, limit);
+        System.out.println("Enter your status *** 0 for single <-> 1 for in relationship <-> 2 for married");
+        EStatus status;
+        try {
+            int statusInput = Integer.parseInt(scanner.nextLine());
+            if (statusInput >= 0 && statusInput <= 2) {
+                status = EStatus.values()[statusInput];
+            } else {
+                System.out.println("Enter from 0 to 2");
+                return;
+            }
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
+            System.out.println("Enter digits only");
+            return;
         }
-        System.out.println("Enter your status(0 for single; 1 for in relationship; 2 for married)");
-        int status = Integer.parseInt(scanner.nextLine());
-        Person newPerson = Person.createPerson(fullName, gender, dob, interest, status);
-        if(personInfo.addNewPerson(newPerson)) {
-            System.out.println("new person added");
-        }else {
-            System.out.println("cannot add person");
+
+        Person newPerson = new Person(name, gender, dateOfBirth, interests, status);
+        if (personService.addNewPerson(newPerson)) {
+            System.out.println("New person added");
+            System.out.println(newPerson.toString());
+        } else {
+            System.out.println("Cannot add person");
         }
     }
 
     private static void removePerson() {
-        System.out.println("Enter existing full name");
-        String fullName = scanner.nextLine();
-        Person existingPerson = personInfo.searchPerson(fullName);
-        if(existingPerson == null) {
-            return;
-        }
-
-        if(personInfo.removePerson(existingPerson)) {
-            System.out.println("person deleted");
-        }else {
-            System.out.println("Something wrong happen");
+        System.out.println("Enter existing personal number uuid");
+        String uuid = scanner.next();
+        UUID uuidInput = UUID.fromString(uuid);
+        Person existingPerson = personService.isPersonCreated(uuidInput);
```

Comment:
Its better to let the service do this kind of check, which means that isPersonCreated should be a private method in the PersonService. I see you already do this check in the remove method so this is not needed here